### DISCUSSION TITLE
Database baseline

### DISCUSSION THREAD

**Discussion by davidmrz** [playframework/playframework#12166] [Q&A]
Hi, does Play have a way to manage baselines for evolutions? 

Something similar to [flyway's](https://documentation.red-gate.com/fd/baseline-184127456.html)

**Comment by mkurz** [comment]
It's all documented: https://www.playframework.com/documentation/latest/Evolutions

There are also (unofficial) play/sbt flyway plugins:
- http://tototoshi.github.io/play-flyway/
  - https://github.com/tototoshi/play-flyway
- https://github.com/flyway/flyway-sbt
